upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1448,6 +1448,11 @@ for chart_name in "${CHARTS[@]}"; do
   do_upgrade_csm_chart "${chart_name}" sysmgmt.yaml
 done
 
+# Ensure kata containers hypervisor file has the list of annotations required for CSM 1.5 CASMTRIAGE-6414
+for node in $(kubectl get nodes | grep -E "^ncn-w[0-9]" | awk '{ print $1 }'); do
+  ssh "${node}" "sed -i 's/\[\"enable_iommu\"\]/[\"enable_iommu\", \"virtio_fs_extra_args\", \"default_memory\", \"kernel_params\", \"cpu_features\"]/ /opt/kata/share/defaults/kata-containers/configuration-qemu.toml'"
```

Comment:
I believe the 2nd single quote should go after sed expression `s///`, not after file name. In it's current form, script errors out:

    [2023-12-15T14:03:36.096Z] sed: -e expression #1, char 115: unknown option to `s'
    [2023-12-15T14:03:36.096Z] 1389 /usr/share/doc/csm/upgrade/scripts/upgrade/prerequisites.sh
    [2023-12-15T14:03:36.096Z] ssh "${node}" "sed -i 's/\[\"enable_iommu\"\]/[\"enable_iommu\", \"virtio_fs_extra_args\", \"default_memory\", \"kernel_params\", \"cpu_features\"]/ /opt/kata/share/defaults/kata-containers/configuration-qemu.toml'"